code/README.md

```diff
@@ -1,8 +1,20 @@
 ## Analyzing the Data
 
-The Makefile contains two commands: `test` and `coverage`. 
+The Makefile contains recipes to perform the intermediate analysis for the project, as well as generate some supplementary work and test all user-defined functions. The results of this directory should largely be regarded as transitional (but nonetheless valuable) steps in the progression of our work. Many of the scripts run processes that are no longer in use, or primarily serve as illustrative examples on how to utilize certain functions for a single subject. For those strictly interested in our final analysis, we recommend looking at our `project-alpha/final` directory instead. 
 
-- `make test`: Tests the functions associated with analyzing the data. 
-- `make coverage`: Generates a coverage report for the functions associated analyzing the data. 
+- `make all`: Performs all intermediate data analysis. Does not include the supplementary work rendered by `make misc`. 
+- `make figs`: Runs only the scripts needed to reproduce figures that are used in the report and slides. Note that these figures is not exhaustive: please navigate to `project-alpha/final` and run `make all` there to reproduce all figures required for the report and slides. 
+- `make misc`: Generates supplementary work. More specifically, it renders an iPython Notebook with early exploratory data analysis into a viewable HTML file in the `utils/misc` subdirectory. 
```

Comment:
@hiroto-udagawa you converted the ipython notebook to a python file right?  wanna talk to Jane about this part?